crates/test/mock-consensus/src/builder/init_chain.rs

```diff
@@ -19,6 +19,11 @@ use {
 
 impl Builder {
```

Comment:
Not in this diff, but there's a typo on the docstring for `penumbra_mock_consensus::builder::Builder`:

> A **buider**, used to prepare and instantiate a new

Reply:
good catch! fixed in f9e29ef :sparkles: